dist/AlphaWallet.js

```diff
@@ -39974,7 +39974,7 @@ var AlphaWallet = {
     var callback = callbacks[id];
 
     if (callback.isRPC) {
-      var response = { 'id': id, jsonrpc: '2.0', result: value, error: { message: error } };
+      var response = { 'id': id, jsonrpc: '2.0', result: value, error: error };
```

Comment:
@oa-s is it better to check `error` if it's just a string and keep the old behavior?

```
if error is string //I forgot my JavaScript
    error: { message: error }
else
    error: error
end
```

Reply:
yes, looks good